tasks/volumes.yml

```diff
@@ -26,6 +26,7 @@
     -b {{ item.backing_image }}
     {% endif %}
   with_items: "{{ volumes }}"
+  when: "{{ item.device == 'disk' }}"
```

Comment:
Currently the device is optional, and defaults to `libvirt_volume_default_device`.